ros2_control_demo_hardware/src/diffbot_system.cpp

```diff
@@ -179,10 +179,11 @@ hardware_interface::CallbackReturn DiffBotSystemHardware::on_deactivate(
   return hardware_interface::CallbackReturn::SUCCESS;
 }
 
-hardware_interface::return_type DiffBotSystemHardware::read()
+hardware_interface::return_type DiffBotSystemHardware::read(
+  const rclcpp::Time & time, const rclcpp::Duration & period)
 {
-  current_timestamp = clock_.now();
-  rclcpp::Duration dt = current_timestamp - last_timestamp_;  // Control period
+  current_timestamp = time;
```

Comment:
Can you please also clean initialization of `clock_` and members `last_timestamp_` from the files. I think they are not needed.

Reply:
done